telegram/_files/inputmedia.py

```diff
@@ -115,13 +115,162 @@ def _parse_thumbnail_input(thumbnail: Optional[FileInput]) -> Optional[Union[str
         )
 
 
+class InputPaidMedia(TelegramObject):
+    """
+    Base class for Telegram InputPaidMedia Objects. Currently, it can be one of:
+
+    * :class:`telegram.InputMediaPhoto`
+    * :class:`telegram.InputMediaVideo`
+
+    .. versionadded:: NEXT.VERSION
+
+    .. seealso:: :wiki:`Working with Files and Media <Working-with-Files-and-Media>`
+
+    Args:
+        media_type (:obj:`str`): Type of media that the instance represents.
+        media (:obj:`str` | :term:`file object` | :obj:`bytes` | :class:`pathlib.Path` | \
+            :class:`telegram.PhotoSize` | :class:`telegram.Video`): File to send. |fileinputnopath|
+            Lastly you can pass an existing telegram media object of the corresponding type
+            to send.
+
+    Attributes:
+        type (:obj:`str`): Type of the input media.
+        media (:obj:`str` | :class:`telegram.InputFile`): Media to send.
+    """
+
+    PHOTO: Final[str] = constants.InputPaidMediaType.PHOTO
+    """:const:`telegram.constants.InputPaidMediaType.PHOTO`"""
+    VIDEO: Final[str] = constants.InputPaidMediaType.VIDEO
+    """:const:`telegram.constants.InputPaidMediaType.VIDEO`"""
+
+    __slots__ = ("media", "type")
+
+    def __init__(
+        self,
+        media_type: str,
```

Comment:
```suggestion
        type: str,
```

tbh I don't know why we deviate from the API in `InputMedia` but I'd like to avoid doing that here …